aragon/app-legacy
# Feat: E2E testing for DAO Creation flow

Task: [APP-1850](https://aragonassociation.atlassian.net/browse/APP-1850)

## Steps to test
- Add your private key to the .env file using the `PRIVATE_KEY` key(Use a separate MetaMask account if possible)/ or any other method specified in Synpress docs
- Run `yarn e2e`
- Cypress window opens up with a controlled browser window that runs through the test cases

Note: The address should have some ETH, otherwise the test would fail. Console warnings would be printed on synpress cmd line, please ignore that.

## Type of change




## Checklist:

- [x] I have selected the correct base branch.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1850]: https://aragonassociation.atlassian.net/browse/APP-1850?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
